src/js/components/help/__snapshots__/help.test.js.snap

```diff
@@ -245,14 +245,14 @@ exports[`Help Component renders correctly 1`] = `
         </h3>
         <p>
           We support several ways of integrating your devices with Mender. For a detailed introduction read the
-           
+
           <a
             class=""
             href="https://docs.mender.io/development/client-installation"
             rel=""
```

Comment:
Interesting, so links to docs.mender.io already didn't have noreferrer?

Reply:
it is possible :)